molecule/provisioner/ansible/plugins/libraries/molecule_vagrant.py

```diff
@@ -294,7 +294,9 @@ def up(self, no_provision=True):
         cd = self._created()
         if not cd:
             changed = True
-            self._vagrant.up(no_provision)
+            for line in self._vagrant.up(no_provision, stream_output=True):
+              # Add prefix to ensure that output of 'vagrant up' doesn't start with one of the JSON start characters { or ]
+              print ('<vagrant_output> {}'.format(line))
```

Comment:
Nice sleuthing.   I'll look this over.  Initially I'm not so sure I Like the idea of prefixing with <vagrant_output>.  However, this only shows up with `--debug` so might not be so bad.

Regardless, very nice investigative work.

Thanks for the feedback and use of Molecule BTW.